pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -718,6 +721,74 @@ const (
 	minSafeProxySize = 2
 )
 
+func (cr *PerconaXtraDBCluster) setProbesDefaults() {
+	if cr.Spec.PXC != nil {
+		if cr.Spec.PXC.LivenessInitialDelaySeconds != nil {
+			cr.Spec.PXC.LivenessProbes.InitialDelaySeconds = *cr.Spec.PXC.LivenessInitialDelaySeconds
+		} else if cr.Spec.PXC.LivenessProbes.InitialDelaySeconds == 0 {
+			cr.Spec.PXC.LivenessProbes.InitialDelaySeconds = 300
+		}
+
+		if cr.Spec.PXC.LivenessProbes.TimeoutSeconds == 0 {
+			cr.Spec.PXC.LivenessProbes.TimeoutSeconds = 5
+		}
+		if cr.Spec.PXC.LivenessProbes.SuccessThreshold > 1 || cr.Spec.PXC.LivenessProbes.SuccessThreshold == 0 {
+			cr.Spec.PXC.LivenessProbes.SuccessThreshold = 1
+		}
```

Comment:
logic have no sense, just put `1` always
```suggestion
		cr.Spec.PXC.LivenessProbes.SuccessThreshold = 1
```
the same for HAProxy

Reply:
oh, yes